iceberg/iceberg-handler/src/test/queries/positive/col_stats.q

```diff
@@ -3,58 +3,102 @@
 set hive.stats.autogather=true;
 set hive.stats.column.autogather=true;
 
+-- Create source table
+drop table if exists src_ice;
+create external table src_ice(
+    a int,
+    b string,
+    c int)
+stored by iceberg;
+
+insert into src_ice values
+    (1, 'one', 50),
+    (2, 'two', 51),
+    (2, 'two', 51),
+    (2, 'two', 51),
+    (3, 'three', 52),
+    (4, 'four', 53),
+    (5, 'five', 54),
+    (111, 'one', 55),
+    (333, 'two', 56);
+
+-- Test hive.iceberg.stats.source = iceberg
 set hive.iceberg.stats.source=iceberg;
+
+
+-- Test NON-PARTITIONED table with hive.iceberg.stats.source=iceberg
 drop table if exists tbl_ice_puffin;
-create external table tbl_ice_puffin(a int, b string, c int) stored by iceberg tblproperties ('format-version'='2');
-insert into tbl_ice_puffin values (1, 'one', 50), (2, 'two', 51),(2, 'two', 51),(2, 'two', 51), (3, 'three', 52), (4, 'four', 53), (5, 'five', 54), (111, 'one', 55), (333, 'two', 56);
-insert into tbl_ice_puffin values (1, 'one', 50), (2, 'two', 51),(2, 'two', 51),(2, 'two', 51), (3, 'three', 52), (4, 'four', 53), (5, 'five', 54), (111, 'one', 55), (333, 'two', 56);
-explain select * from tbl_ice_puffin order by a, b, c;
-select * from tbl_ice_puffin order by a, b, c;
```

Comment:
The `SELECT *` and `ORDER BY` was replaced by different kind of queries. Are we sure that we are not losing test coverage with respect to the original purpose of the test?